app/models/google_calendar/google_event.rb

```diff
@@ -0,0 +1,18 @@
+module GoogleCalendar
+  class GoogleEvent
+    extend Listing
+    extend Adding
+    extend Deleting
+
```

Comment:
:) Why do you want to use mixin facility. Mixin is type of inheritance. Do you want GoogleEvent inherits from Listing, Adding and Deleting or GoogleEvent should be able to listing, adding and deleting? Whats more Listing module looks more like set of public methods.